plugins/TimingTriggerCandidateMaker.cpp

```diff
@@ -129,7 +134,13 @@ TimingTriggerCandidateMaker::receive_hsievent(dfmessages::HSIEvent& data)
   }
 
   ++m_tsd_received_count;
-  
+
+  if (m_prescale_flag) {
+    if (m_tsd_received_count % m_prescale != 0){
```

Comment:
Is this the correct logic? Shouldn't it be `if (m_tsd_received_count % m_prescale == 0)`?

Reply:
Not in this case, if you look at what follows this if, it's `return`; meaning in the case of `==0` we would go on and continue processing the hsi event, creating TC, and sending it; while in the case of `!=0` we simply skip the whole processing and wait for another hsi event.